client/src/components/Header.css

```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
```

Comment:
These imports should be at a higher level if possible, e.g. in the index.html, as they will be used in other components

Reply:
that makes sense, moved